app/src/main/kotlin/games/TicTacToe.kt

```diff
@@ -8,19 +8,20 @@ class TicTacToe(
         val player2: Player = Player(name = "Paul", id = 2)
 ) {
     private val greeting = "Welcome to the Tic Tac Toe game!"
-    private val board = Board()
+    internal val board = Board()
     private val displayer = TextDisplayer()
+    private var winner:Player? = null
 
     //    /// Print the updated board after every move
-    fun printBoard(board: Board) {
+    fun printBoard() {
```

Comment:
There was no point passing in the board when it is defined as part of the class.